src/main/java/frc/robot/RobotContainer.java

```diff
@@ -89,13 +105,21 @@ private void configureButtonBindings() {
       OI.Driver.getOuttakeButton().whileTrue(new StartEndCommand(intakeSubsystem::push, intakeSubsystem::stop, intakeSubsystem));
     }
 
+    if (Robot.armEnabled) {
+      OI.Driver.getArmHighButton().toggleOnTrue(new StartEndCommand(() -> armSubsystem.transitionToState(ArmState.HIGHSCORE).schedule(), () -> armSubsystem.transitionToState(ArmState.STOWED).schedule()));
+      OI.Driver.getArmMidButton().toggleOnTrue(new StartEndCommand(() -> armSubsystem.transitionToState(ArmState.MIDSCORE).schedule(), () -> armSubsystem.transitionToState(ArmState.STOWED).schedule()));
+      OI.Driver.getArmIntakeButton().toggleOnTrue(new StartEndCommand(() -> armSubsystem.transitionToState(ArmState.CONEINTAKE).schedule(), () -> armSubsystem.transitionToState(ArmState.STOWED).schedule()));
+    }
```

Comment:
Not sure the desired behavior here, but [Chaining Calls](https://docs.wpilib.org/en/stable/docs/software/commandbased/binding-commands-to-triggers.html#chaining-calls) might a better method. Behavior would then be hold the button -> move to state, then release the button -> back to stowed

Reply:
I changed how these controls behave, it uses proxy commands now